pages/common/highlight.md

```diff
@@ -0,0 +1,28 @@
+# highlight
+
+> Outputs syntax-highlighted source code to a variety of formats.
+> More information: <http://www.andre-simon.de/doku/highlight/en/highlight.php>.
+
+- Produce a complete HTML document from a source code file:
+
+`highlight -O html --style {{theme name}} --syntax={{language}} {{source file}}`
+
+- Produce an HTML fragment, suitable for inclusion in a larger document:
+
+`highlight -O html --fragment --syntax={{language}} {{source file}}`
+
+- Inline all CSS styling in every tag:
+
+`highlight -O html --inline-css --syntax={{language}} {{source file}}`
+
+- List all supported languages:
+
+`highlight --list-scripts lang`
+
+- List all installed syntax highlighting themes:
+
+`highlight --list-scripts themes`
```

Comment:
```suggestion
`highlight --list-scripts {{themes}}`
```

Reply:
This is also a wrong suggestion because `themes` is also a keyword here, not a variable.